rust/connlib/tunnel/src/client.rs

```diff
@@ -119,10 +119,8 @@ pub struct ClientState {
 
     /// Maps from connlib-assigned IP of a DNS server back to the originally configured system DNS resolver.
     dns_mapping: BiMap<IpAddr, DnsServer>,
-    /// DNS queries that had their destination IP mangled because the servers is a CIDR resource.
-    ///
-    /// The [`Instant`] tracks when the DNS query expires.
-    mangled_dns_queries: HashMap<(SocketAddr, u16), Instant>,
+    /// UDP DNS queries that had their destination IP mangled to redirect them to another DNS resolver through the tunnel.
```

Comment:
Out of curiosity, why is the TCP equivalent not needed?

Reply:
The TCP equivalent is defined on line 136.